GitHub-Test-DHprojects/Gardiner_GitTest4.md

```diff
@@ -0,0 +1,9 @@
+# Joey Gardiner Git Test 4
+
+This project team originally sought to find what types of `obsolete` words were used in Lovecraft's writing. So the team shifted their focus to finding what words Lovecraft used to elicit a feeling of **confusion, ambiguity, and dread.** They measured the terms by categories of `scientific, religious, gothic, or other`.  
+
+
+I thought this was a very thoughtful way to go about searching for what about Lovecraft's writing evokes **uneasiness**. This project helps pinpoint what makes Lovecraft's writing to interesting and well-written. It can help writers that want to create a piece with the type of cosmic or psychological horror Lovecraft writes, take a deeper look into the text and really find what provokes such feelings so they can *utilize similar ideas*.
+
+
+This project gives me an idea of similar regard. My idea focuses on the original trilogy of *Star Wars* and what makes the world seem so realistic. The complication with this idea is it would be looking at both visual and spoken aspects. The way everything is presented allows it to feel lived in, unlike other sci-fi. Developing this idea into something meaningful would be a challenge.
```

Comment:
George Lucas did a fantastic job with world-building despite his financial limitations (for ANH at least) and the limitations of special effects at the time. A large part of his world building comes from characters dialogue and interactions with each other in a natural way while still sprinkling in foreign ideas for the viewer.